build/docker/package.arm64.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+set -euxo pipefail
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
+VERSION=1.26.2
+BUILD_TYPE=${buildConfiguration:-Debug}
+
+mkdir -p $DIR/../../deploy/linux
+cp $DIR/../../integrations.json $DIR/../../src/Datadog.Trace.ClrProfiler.Native/bin/${BUILD_TYPE}/x64/
+cp $DIR/../../build/artifacts/createLogPath.sh $DIR/../../src/Datadog.Trace.ClrProfiler.Native/bin/${BUILD_TYPE}/x64/
+
+# If running the unified pipeline, copy managed assets now instead of in the profiler build step
+if [ -n "${UNIFIED_PIPELINE-}" ]; then
+  mkdir -p $DIR/../../src/Datadog.Trace.ClrProfiler.Native/bin/${BUILD_TYPE}/x64/netstandard2.0
+  cp $DIR/../../src/bin/windows-tracer-home/netstandard2.0/*.dll $DIR/../../src/Datadog.Trace.ClrProfiler.Native/bin/${BUILD_TYPE}/x64/netstandard2.0/
+
+  mkdir -p $DIR/../../src/Datadog.Trace.ClrProfiler.Native/bin/${BUILD_TYPE}/x64/netcoreapp3.1
+  cp $DIR/../../src/bin/windows-tracer-home/netcoreapp3.1/*.dll $DIR/../../src/Datadog.Trace.ClrProfiler.Native/bin/${BUILD_TYPE}/x64/netcoreapp3.1/
+fi
+
+cd $DIR/../../deploy/linux
+for pkgtype in $PKGTYPES ; do
+    fpm \
+        -f \
+        -s dir \
+        -t $pkgtype \
+        -n datadog-dotnet-apm \
+        -v $VERSION \
+        $(if [ $pkgtype != 'tar' ] ; then echo --prefix /opt/datadog ; fi) \
+        --chdir $DIR/../../src/Datadog.Trace.ClrProfiler.Native/bin/${BUILD_TYPE}/x64 \
+        netstandard2.0/ \
+        netcoreapp3.1/ \
+        Datadog.Trace.ClrProfiler.Native.so \
+        integrations.json \
+        createLogPath.sh
+done
+
+gzip -f datadog-dotnet-apm.tar
+
+if [ -z "${MUSL-}" ]; then
+  mv datadog-dotnet-apm.tar.gz datadog-dotnet-apm-$VERSION.arm64.tar.gz
```

Comment:
Is the `.arm64` name the only difference between this and `package.sh`? If so, can we just parameterize this?

Reply:
Yes, right now it is the only difference, ok I will work with the existing one.

Reply:
done in 9761a3b

Reply:
FWIW, I've done a lot of consolidation in the Nuke pipeline, trying to remove a lot of the duplication. I'll need to reimplement all this in Nuke I think

Reply:
I changed to use the same `packages.sh` file in, I don't think is a big change to port if you already have it working on the previous file version. The change is in 9761a3b